Header header
float32[] x
float32[] y
float32[] z
int32[] array_sizes
float32 scan_time
